src/languages/en.ts

```diff
@@ -817,6 +817,28 @@ export default {
         availableSpend: 'Remaining spending power',
         virtualCardNumber: 'Virtual card number',
         physicalCardNumber: 'Physical card number',
+        getPhysicalCard: 'Get physical card',
+    },
+    getPhysicalCard: {
```

Comment:
@grgia 

Can I get this translated to Spanish, please?

Reply:
bump @grgia

Reply:
We might have some of these already and could move them to `common`

https://github.com/Expensify/App/blob/9e35d91dd59b9233a584150a9a19f1ef458e9622/src/languages/en.ts#L1044-L1046